frontend/lib/verification/view/verification_result.dart

```diff
@@ -52,13 +52,18 @@ class VerificationResult extends StatelessWidget {
                 "Verbindung zum Server fehlgeschlagen.", "connectionError"));
           }
           QueryResult verifyResult = snapshot.data[0];
+          if (verifyResult.hasException) {
+            debugPrint(verifyResult.exception.toString());
+            return NegativeVerificationResult(
+                VerificationError("Fehler bei der Kommunikation mit dem "
+                    "Server.", "verifyRequestError"));
+          }
           QueryResult regionsResult = snapshot.data[1];
-          if (verifyResult.hasException || regionsResult.hasException) {
-            var exception = verifyResult.hasException
-                ? verifyResult.exception.toString()
-                : regionsResult.exception.toString();
+          if (regionsResult.hasException) {
+            debugPrint(regionsResult.exception.toString());
             return NegativeVerificationResult(
-                VerificationError(exception, "verifyRequestError"));
+                VerificationError("Fehler bei der Kommunikation mit dem "
+                    "Server.", "queryRegionsError"));
           }
```

Comment:
Has nothing to do with the bug, but I think we should not just forward potentially cryptic exception messages to the user but instead log them.